chore: update fs.l/statSync API history for throwIfNoEntry (#36882)
nodejs/node: doc/api/fs.md

<!-- YAML
added: v0.1.30
changes:
  - version: v15.3.0
    pr-url: https://github.com/nodejs/node/pull/33716

>     description: Accepts an additional `options` object to specify whether

Reviewer: ```suggestion
    description: Accepts a `throwIfNoEntry` option to specify whether
```
because the `options` object already exists at this point.

Ditto with the changes below.

Author: Oops, I read that as "an additional `options` property".
